changes/ticket32604

```diff
@@ -0,0 +1,7 @@
+  o Minor features (onion services):
+    - Version 3 onion services can now use HiddenServiceExportRendPoint
+      to get first 12 chars of the rendezvous point's fingerprint.
+    - Version 3 onion services can now use HiddenServiceExportInstanceID
+      which enables them to differenciate traffic when multiple tor
```

Comment:
```suggestion
      to differentiate traffic when multiple tor
```

Reply:
Fixed in `2582e28`.

Reply:
I think it's probably easiest if I fix the remaining spelling mistake in the changes file?

Reply:
Fixed in 3a8dee4.